### PR TITLE
PHP Inspections changes

### PR DESCRIPTION
**Pull request by gulaandrij**

### CODE REVIEW

**Comment by gonzakpo** [comment]
why use \is_array ?

**Comment by gulaandrij** [comment]
Using '\func name' would enable some of opcode optimizations.
Inspection info: Using fully-qualified function calls is faster, details and benchmarks are here
https://github.com/Roave/FunctionFQNReplacer

**Comment by gulaandrij** [comment]
test falling in 7.0 php, maybe we can drop it for master and keep on 1.2.x?

**Comment by gulaandrij** [comment]
or should I revert changes?

**Comment by gonzakpo** [comment]
- phpstan/phpstan 0.10.3 requires php ~7.1 -> your PHP version (7.0.32) does not satisfy that requirement.

**Comment by alterphp** [comment]
Branch 1.x of this budle must support PHP 7.0. Please keep it in mind for static analysis too.

**Comment by gulaandrij** [comment]
I reverted changes. Can it be merged now? @alterphp

File: src/Configuration/EmbeddedListViewConfigPass.php [hunk 1]
```diff
@@ -56,11 +56,11 @@ private function processSortingConfig(array $backendConfig)
                 $backendConfig['entities'][$entityName]['embeddedList']['sort'] = $entityConfig['list']['sort'];
             } elseif (isset($entityConfig['embeddedList']['sort'])) {
```
Comment (alterphp) [inline src/Configuration/EmbeddedListViewConfigPass.php]: Why `isset` and `strtoupper` do not require explicit namespace Same for array_key_exists, etc

File: src/Configuration/ShowViewConfigPass.php [hunk 2]
```diff
@@ -20,7 +24,12 @@ class ShowViewConfigPass implements ConfigPassInterface
         'embedded_list' => '@EasyAdminExtension/default/field_embedded_list.html.twig',
     );
 
-    public function __construct($embeddedListHelper)
+    /**
```
Comment (alterphp) [inline src/Configuration/ShowViewConfigPass.php]: I don't like type hinting with class, it reduces extensivity. The best approach, ihmo, is to type hint with interfaces. For now, project is too young to define those interfaces, so I prefer no type hint at all for services constructor arguments.

Please revert those changes.

**Comment by alterphp** [comment]
> I reverted changes. Can it be merged now? @alterphp

I'll merge it next week if it's ok.

File: .gitignore [hunk 3]
```diff
@@ -4,3 +4,4 @@
 /composer.lock
 /docker-compose.yml
 /phpunit.xml
+.idea
```
Comment (alterphp) [inline .gitignore]: Please revert this too (see https://insight.symfony.com/projects/a7179df5-4ed7-468c-899c-891535dbe802/analyses/231?status=commit) and leave a blank line at the end of file.

**Comment by gulaandrij** [comment]
done

**Comment by gonzakpo** [comment]
It is not so simple to merge,  it took me 76 commits. jajaja

File: src/Configuration/EmbeddedListViewConfigPass.php (hunk 1 above)
Comment (gulaandrij) [reply src/Configuration/EmbeddedListViewConfigPass.php]: isset cant be write like this

File: src/Configuration/ShowViewConfigPass.php (hunk 2 above)
Comment (gulaandrij) [reply src/Configuration/ShowViewConfigPass.php]: reverted

File: .php_cs [hunk 4]
```diff
@@ -24,5 +24,6 @@ return PhpCsFixer\Config::create()
         'php_unit_strict' => true,
         'phpdoc_summary' => false,
         'strict_comparison' => false,
+        'native_function_invocation'=> true
```
Comment (alterphp) [inline .php_cs]: :+1:

**Comment by alterphp** [comment]
Thanks @gulaandrij for improvements !

What tools did you use and what config ?

**Pull request merged**